_topic_maps/_topic_map.yml

```diff
@@ -183,8 +183,13 @@ Distros: openshift-acs
 Topics:
 - Name: Viewing the dashboard
   File: view-dashboard
-- Name: Managing compliance
-  File: manage-compliance
+- Name: Managing Compliance
+  Dir: manage-compliance
+  Topics:
+  - Name: Managing compliance (1.0)
+    File: managing-compliance-10
+  - Name: Managing compliance (2.0)
+    File: managing-compliance-20
 - Name: Using the Compliance Operator
```

Comment:
I wonder if we want to bump the compliance operator installation section up above the "managing" sections since it required in both 1.0 and 2.0 compliance flows.

For the compliance operator install docs themselves, I believe @sheriff-rh made them modular so it's easier to include relevant sections from other books without duplicating content.

https://docs.openshift.com/container-platform/4.14/security/compliance_operator/co-management/compliance-operator-installation.html

Reply:
@dashrews78 WDYT?

Reply:
I think this is a good idea.  However it got me thinking about our existing docs.  In the ACS docs where we mention creating a `ScanSettingBinding`.  For 2.0 that is no longer required and not preferred.  So we should probably mention something to that effect if using 2.0 that the Scan Schedule feature can be used instead of creating the ScanSettingBinding on the compliance operator directly.

https://71171--docspreview.netlify.app/openshift-acs/latest/operating/manage-compliance-operator/compliance-operator-rhacs#compliance-operator-configure-scanning_compliance-operator-rhacs